client/src/components/layouts/CollaborationRequest.tsx

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import Media from 'react-bootstrap/esm/Media';
+import Container from 'react-bootstrap/esm/Container';
+import Row from 'react-bootstrap/esm/Row';
+import Col from 'react-bootstrap/esm/Col';
+import Button from 'react-bootstrap/esm/Button';
+
+
+function CollaborationRequest(props: any) {
+
+ function HandlerAccepter(){
+
+    console.log(`${props.username} " a été ajouté au "${props.document}" en tant que collaborateur"`)
+  }
+
+  function HandlerRefuse(){
+
+    console.log(`la demande de "${props.username}" a été refusée"`)
+  }
+    return (
+
+ <Container>
+  <Row>
+  <Media>
+  <img
```

Comment:
https://react-bootstrap.github.io/components/images/

Reply:
d'accord

Reply:
I think it's done